### DISCUSSION TITLE
EIP-1193 is not custom it is standard.

### DISCUSSION THREAD

**Discussion by pociej** [wevm/viem#1368] [General]
Why `viem` is designed that way if you want to play with well standarized EIP-1193 you have to use same `custom` transport as when you are going to really `custom` implementation where you define `request`. While it is working it looks really misleading.

**Comment by jxom** [comment]
I am not sure what you mean. You bring your own custom EIP-1193 shaped request function.

**Reply by pociej** [reply]
you bring or you do not. typical usage is `custom(window.ethereum)` I would leave custom for cases where you really specify your own request.
